modules/installation-dns-bare-metal.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+[id="installation-dns-bare-metal-{context}"]
+= Bare metal DNS requirements
+
+The following DNS records are required for an {product-title} cluster. In each
+record, `<cluster_name>` is the cluster name and `<base_domain>` is the cluster
+base domain that you specify in the `install-config.yaml` file.
+
+.Required DNS records
+[cols="1a,3a,5a",options="header"]
+|===
+
+|Component
+|Record
+|Description
+
+|Kubernetes API
+|`api.<cluster_name>.<base_domain>`
+|This DNS record must point to the load balancer
+for the control plane machines. This record must be resolvable by both clients
+external to the cluster and from all the nodes within the cluster.
+
+|Routes
+|`*.apps.<cluster_name>.<base_domain>`
+|A wildcard DNS record that points to the load balancer that targets the
+machines that  run the ingress router pods, which are the worker nodes by
+default. This record must be resolvable by both clients external to the cluster
+and from all the nodes within the cluster.
+
+.2+|Etcd
+|`etcd-<idx>.<cluster_name>.<base_domain>`
```

Comment:
@abhinavdahiya, what is IDX here?

Reply:
`idx` is the control plane host number (starting at `0`)
eg: if the user setup 3 control plane hosts like:

[install-config]
```yaml
apiVersion: v1beta3
controlPlane:
  name: master
  replicas: 3
...
```
`etcd-0.<cluster_name>.<base_domain>` points to 1st control plane host
`etcd-1.<cluster_name>.<base_domain>` points to 2nd control plane host
`etcd-2.<cluster_name>.<base_domain>` points to 3rd control plane host

Reply:
Ah, idx = index. Thank you!